docs/index.rst

```diff
@@ -30,6 +30,7 @@ User Documentation
 
```

Comment:
In line#5 of index.rst there is a typo. It states:
``pahfit`` is a python package to is a decomposition model  ... 
which should be replace by sth like:
``pahfit`` is a python package containing a decomposition model

Reply:
Fixed.